istio/grafana-virtual-service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: grafana-vs
+  namespace: kubeflow
```

Comment:
Is VirtualService namespace scoped or cluster-wise?

Reply:
namespaced

Reply:
So should we make namespace a parameter or ```kubeflow``` should be good for most cases?

Reply:
I think kubeflow is good for now.